src/dataServices/dynamoDbDataService.ts

```diff
@@ -163,14 +158,13 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
             resource: resourceCopy,
         };
 
-        let item: any = {};
         // Sending the request to `atomicallyReadWriteResources` to take advantage of LOCKING management handled by
         // that method
         const response: BundleResponse = await this.transactionService.transaction({
             requests: [batchRequest],
             startTime: new Date(),
         });
-        item = clone(resource);
+        const item = clone(resource);
         const batchReadWriteEntryResponse = response.batchReadWriteResponses[0];
         item.meta = generateMeta(batchReadWriteEntryResponse.vid, new Date(batchReadWriteEntryResponse.lastModified));

```

Comment:
Does this explicitly need to happen or is the `meta`  returned from the bundler if we can remove this part I'd feel better

Reply:
For Update requests the Bundler returns an empty `resource` attribute. However, it does return the fields that are needed to regenerate the `meta` attribute.

The thought being for Updates, as long as an error isn't thrown the resource that was provided to the Bundler is the resource that is written to DDB. Only changes being `vid` and the updated timestamp.